types/paper/index.d.ts

```diff
@@ -1247,7 +1278,7 @@ declare module paper {
         /**
          * only hit-test again a certain item class and its sub-classes: Group, Layer, Path, CompoundPath, Shape, Raster, PlacedSymbol, PointText, etc.
          */
-        class?: string;
+        class?: Function;
```

Comment:
prefer `(...args: any[]) => any` to `Function` unless you want to support functions created with `new Function(...)`.

Reply:
Changed to `class?: { new(...args: any[]): Item; };`